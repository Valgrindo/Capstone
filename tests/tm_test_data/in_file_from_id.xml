<!-- Test that a component  defined separately from a command still resolves correctly. -->
<commands>

    <command name="TO_RESOLVE">
        <component indicator="SPEECHACT" type="SA_TELL">
            <role name="CONTENT">
                <!-- This component sis defined externally -->
                <component from_id="ID"/>
            </role>
        </component>
    </command>

    <!-- And here is the definition -->
    <component id="ID" indicator="BARE" type="FRUIT" word="APPLE"/>

</commands>