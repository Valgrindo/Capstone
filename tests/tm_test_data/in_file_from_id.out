<command TO_RESOLVE>
<component SPEECHACT SA_TELL>
|  <rolegroup>
|  |  <role CONTENT>
|  |  |  <component BARE FRUIT APPLE>
|  </rolegroup>
</command>